### PR TITLE
feat: create new table from import

### PR DESCRIPTION
**Pull request by luka-nextcloud**
* Resolves: #836

[demo.webm](https://github.com/nextcloud/tables/assets/89908051/9dacf4bf-88d0-457d-b280-6872af271501)

### CODE REVIEW

**Comment by juliushaertl** [comment]
@luka-nextcloud Feel free to also involve others for review ;)

File: cypress/e2e/tables-table.cy.js [hunk 1]
```diff
@@ -23,13 +23,35 @@ describe('Manage a table', () => {
...
+		cy.uploadFile('test-import.csv', 'text/csv')
+		cy.contains('.app-menu-entry--label', 'Tables').click()
+		cy.contains('button', 'Create new table').click()
+		cy.get('.tile').contains('Import').click({ force: true })
+		cy.get('.modal__content').should('be.visible')
+		cy.get('.modal__content input[type="text"]').clear().type('import list')
+		cy.contains('button', 'Create table').scrollIntoView().click()
+		cy.contains('h2', 'Import').should('be.visible')
+
+		cy.get('.modal__content button').contains('Select a file').click()
```
Comment (max-nextcloud) [inline cypress/e2e/tables-table.cy.js:43]: Failing cypress test indicates that this is not `Select a file` but `Select from Files`. I think it actually should be `Select from files` as we only capitalize the first word.

image

**Review by max-nextcloud — COMMENTED** [review]
Nice that this includes cypress tests. Makes the review much easier.

Please fix them and add the screenshots for design review as proposed by julius.

**Review by juliushaertl — APPROVED** [review]

**Comment by luka-nextcloud** [comment]
Uploaded a video recording.

**Pull request merged**